espnet/bin/asr_recog.py

```diff
@@ -173,14 +177,18 @@ def main(args):
             from espnet.asr.chainer_backend.asr import recog
             recog(args)
         elif args.backend == "pytorch":
-            # Experimental API that supports custom LMs
-            if args.api == "v2":
-                from espnet.asr.pytorch_backend.recog import recog_v2
-                recog_v2(args)
+            if args.num_encs == 1:
```

Comment:
If you wil skip implementing V2 AP for multi enc in this PR, `raise NotImplementedError(f"--num-encs {args.num_encs} > 1 is not supported in --api v2")` at this time.

Reply:
I will change it!